docs/source/contribution.rst

```diff
@@ -142,6 +142,27 @@ Please note the followings when writing the document.
   the document.
 
 
+Building Guidelines
+-------------------
+
+In order to run unit tests at the repository root, you first have to build Cython files in place by running the following command::
+
+  $ python setup.py develop
+
+Please note that when you modify ``*.pxd`` files, you must clean ``*.cpp`` and ``*.so`` files once with::
```

Comment:
@niboshi I am not familiar with sphinx, and I could not find how to write a preformatted text in `.. note::` section.  Do you know it?